openmxparser/parser.py

```
@@ -104,9 +106,16 @@ def parse(self, mainfile: str, archive: EntryArchive, logger):
         # FIXME: add some testcase for DFT+U
         scf_hubbard_u = mainfile_parser.get('scf_hubbard_u')
         if scf_hubbard_u is not None:
-            if mainfile_parser.get('scf_hubbard_u').lower == 'on':
```

This is a nice catch of a bug introduced by me. But this goes specifically into the category of stuff that should have its own commit and not go into the unrelated XC_functional_name commit.

---

I found this syntax error and tried to solve it but i forgot that it will be automatically added within this commit. so i am going to ignore this error with new pull request.

---

Next time you can use git add --interactive to add only parts of changes in file to the commit